filebeat/module/kibana/audit/config/audit.yml

```diff
@@ -0,0 +1,16 @@
+type: log
+paths:
+{{ range $i, $path := .paths }}
+ - {{$path}}
+{{ end }}
+exclude_files: [".gz$"]
+
+processors:
+  - add_locale: ~
+  - add_fields:
+      target: ''
+      fields:
+        ecs.version: 1.6.0
```

Comment:
Is this something we should log from Kibana instead? 
If we want to update to a later version of ECS that should be controlled by Kibana, not the Filebeat module.

Reply:
Good question; I think this would make sense for Kibana to log instead of Filebeat. I copied this over from the ES config, but it probably doesn't make sense for us to follow suit here.

Reply:
Cool, I'll create a PR to log that in Kibana

Reply:
I came across this when testing. It appears that Filebeat is expecting that it'll set an explicit version itself:

https://github.com/elastic/beats/blob/4dd8061938e8aee474ef6e378fe9aae2bd4837da/filebeat%2Ftests%2Fsystem%2Ftest_modules.py#L104

I worked around this, but I don't know if that's "ok" or not 😕